### DISCUSSION TITLE
Lodestar Standup 2024-22 (June 11, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#6855] [Standups]
### Meeting Info
[June 11, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-china-shanghai-japan-tokyo-australia-sydney/jun-11-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
- Planning Items
  - v1.19.0 Release
  - v1.20.0 Planning
- Team Updates
- Open Discussion/Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by philknows** [comment]
Please have a look at the proposed testing infrastructure changes and bring any discussion points about it to standup, thanks!

**Comment by twoeths** [comment]
need to discuss about v1.19.0 deployments noting the `nHistoricalStates` flag we want to test
also proposer boost reorg test plan

**Comment by philknows** [comment]
Closing for #6893
